core/src/pluginmanager.cpp

```diff
@@ -133,8 +133,10 @@ Plugin* PluginManager::loadPlugin(QString file)
 
 	QPluginLoader qp(file);
 	ret = qp.load();
-	if(!ret)
-		return nullptr;
+	if(!ret) {
+                qWarning(CAT_PLUGINMANAGER) << qp.errorString();
```

Comment:
spaces vs tabs 
also, you can remove this, I added it here c463060b081a31e30e374e9c44e319eb3a0eea5f